testsuite/cluster-test/src/main.rs

```diff
@@ -48,7 +48,7 @@ use tokio::runtime::Runtime;
 const HEALTH_POLL_INTERVAL: Duration = Duration::from_secs(5);
 
 #[derive(StructOpt, Debug)]
-#[structopt(group = ArgGroup::with_name("action").required(true))]
```

Comment:
We check to make sure changelog.len == 2 in line 207 below. Maybe we should keep the required?

Reply:
Oh, but we check it on `changelog` parameter, not `action` group.
I removed `required` here because someone may want to just use `--deploy` or `--changelog` without running test